### DISCUSSION TITLE
Mail scored 34.23 /14 but no action

### DISCUSSION THREAD

**Discussion by Oxyash** [rspamd/rspamd#5228] [Q&A]
after update to  3.10.2 version debian 11
some of mail donts take any action even they get scored for spam 34.23 /14
but its not  allways
waht can be wrong ?

URIBL_BLACK (7) [apandabas.info:url]
ABUSE_SURBL (5.5) [apandabas.info:url]
BAYES_SPAM (5.099259) [99.99%]
MX_MISSING (3.5) []
RBL_SPAMHAUS_SBL (2) [165.84.217.32:from]
RBL_SPAMHAUS_CSS (2) [165.84.217.32:from]
MISSING_MIME_VERSION (2)
MIME_HEADER_CTYPE_ONLY (2)
SUBJ_EXCESS_BASE64 (1.5)
R_BAD_CTE_7BIT (1.05) [7bit,utf8]
HFILTER_HELO_2 (1) [cloudhost-10356613.uk-south-2.nxcli.net]
AUTH_NA (1)
FORGED_SENDER (0.3) [xs37s6og@digital-security-help.org,postmaster@aa5676f869.nxcli.io]
MIME_HTML_ONLY (0.2)
ONCE_RECEIVED (0.1)
MX_GOOD (-0.01) []
HAS_LIST_UNSUB (-0.01)
RCVD_COUNT_ZERO (0) [0]
TO_DN_NONE (0)
RSPAMD_URIBL (0) [apandabas.info:url]
ARC_NA (0)
TO_MATCH_ENVRCPT_ALL (0)
R_DUMMY (0)
ARC_SIGNED (0) [grupaartemis.pl:s=default:i=1]
ASN (0) [asn:31463, ipnet:165.84.216.0/21, country:GB]
FROM_HAS_DN (0)
R_SPF_NA (0) [no SPF record]
FROM_NEQ_ENVFROM (0) [xs37s6og@digital-security-help.org,postmaster@aa5676f869.nxcli.io]
HAS_X_POS (0)
RCPT_COUNT_ONE (0) [1]
R_DKIM_NA (0)
REPLYTO_DOM_NEQ_TO_DOM (0)
MISSING_XM_UA (0)
REPLYTO_DOM_NEQ_FROM_DOM (0)
DMARC_NA (0) [digital-security-help.org]
MIME_TRACE (0) [0:~]
HAS_REPLYTO (0) [no-reply@example.com]

<html><body>

  | no action |  
-- | -- | --

</body>
</html>

**Comment by moisseev** [comment]
1. Don't open issues for generic support questions.

> waht can be wrong ?

2. The answer is likely in the log.  
3. The `R_DUMMY` symbol looks suspicious, as it suggests a user-configured rule.

**Comment by Oxyash** [comment]
thank you 

that the log 

2024-11-20 07:03:11 #453522(rspamd_proxy) <f235a0>; milter; rspamd_milter_process_command: got connection from 165.84.217.32:34722
2024-11-20 07:03:11 #453524(normal) <1aaa3a>; task; rspamd_worker_body_handler: accepted connection from 127.0.0.1 port 36642, task ptr: 00007FF1AF291098
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; rspamd_mime_part_get_cte: detected missing CTE for part as: 7bit
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; rspamd_message_parse: loaded message; id: <20241120060310.14025.qmail@cloudhost-10356613.uk-south-2.nxcli.net>; queue-id: <1C604C2DF4>; size: 2700; checksum: <a2f4ddeee418f2a5b02220e4a5016f28>
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; rspamd_mime_part_detect_language: detected part language: en
2024-11-20 07:03:11 #453524(normal) <1C604C>; lua; settings.lua:391: <20241120060310.14025.qmail@cloudhost-10356613.uk-south-2.nxcli.net> apply static settings global_wblist-14 (id = 3123186775); rcpt matched; priority high
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action reject due to settings
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action greylist due to settings
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action rewrite subject due to settings
2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action add header due to settings
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: RCVD_IN_DNSWL(541): 756.96 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: SEM_URIBL_UNKNOWN(563): 300.96 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: SEM_URIBL_FRESH15_UNKNOWN(558): 308.96 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: RBL_NIXSPAM(561): 316.96 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: SURBL_MULTI(532): 384.96 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; rspamd_check_group_score: maximum group score 12.50 for group surbl has been reached, reduce weight of symbol URIBL_BLACK from 7.50 to 7.00
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: RBL_SENDERSCORE_REPUT_UNKNOWN(519): 595.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; rspamd_spf_maybe_return: not stored SPF record for aa5676f869.nxcli.io (0xa4aa40bbeec59e2b) in LRU cache; flags=4; ttl=0
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: SPF_CHECK(593): 1367.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: HFILTER_CHECK(385): 1335.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: RBL_VIRUSFREE_UNKNOWN(548): 659.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: RBL_SEM(497): 671.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: MX_INVALID(448): 1447.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; finalize_item: slow asynchronous rule: URIBL_MULTI(477): 903.01 ms; no idle timer is needed
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; rspamd_task_write_log: id: <20241120060310.14025.qmail@cloudhost-10356613.uk-south-2.nxcli.net>, qid: <1C604C2DF4>, ip: 165.84.217.32, from: <postmaster@aa5676f869.nxcli.io>, (default: F (no action): [34.22/14.00] [URIBL_BLACK(7.00){apandabas.info:url;},ABUSE_SURBL(5.50){apandabas.info:url;},BAYES_SPAM(5.09){99.99%;},MX_MISSING(3.50){},MIME_HEADER_CTYPE_ONLY(2.00){},MISSING_MIME_VERSION(2.00){},RBL_SPAMHAUS_CSS(2.00){165.84.217.32:from;},RBL_SPAMHAUS_SBL(2.00){165.84.217.32:from;},SUBJ_EXCESS_BASE64(1.50){},R_BAD_CTE_7BIT(1.05){7bit;utf8;},AUTH_NA(1.00){},HFILTER_HELO_2(1.00){cloudhost-10356613.uk-south-2.nxcli.net;},FORGED_SENDER(0.30){xs37s6og@digital-security-help.org;postmaster@aa5676f869.nxcli.io;},MIME_HTML_ONLY(0.20){},ONCE_RECEIVED(0.10){},HAS_LIST_UNSUB(-0.01){},MX_GOOD(-0.01){},ARC_NA(0.00){},ARC_SIGNED(0.00){:s=default:i=1;},ASN(0.00){asn:31463, ipnet:165.84.216.0/21, country:GB;},DMARC_NA(0.00){digital-security-help.org;},FROM_HAS_DN(0.00){},FROM_NEQ_ENVFROM(0.00){xs37s6og@digital-security-help.org;postmaster@aa5676f869.nxcli.io;},HAS_REPLYTO(0.00){no-reply@example.com;},HAS_X_POS(0.00){},MIME_TRACE(0.00){0:~;},MISSING_XM_UA(0.00){},RCPT_COUNT_ONE(0.00){1;},RCVD_COUNT_ZERO(0.00){0;},REPLYTO_DOM_NEQ_FROM_DOM(0.00){},REPLYTO_DOM_NEQ_TO_DOM(0.00){},RSPAMD_URIBL(0.00){apandabas.info:url;},R_DKIM_NA(0.00){},R_DUMMY(0.00){},R_SPF_NA(0.00){no SPF record;},TO_DN_NONE(0.00){},TO_MATCH_ENVRCPT_ALL(0.00){}]), len: 2700, time: 1738.043ms, dns req: 28, digest: <a2f4ddeee418f2a5b02220e4a5016f28>, rcpts: >, settings_id: global_wblist-14
2024-11-20 07:03:12 #453524(normal) <1C604C>; task; rspamd_protocol_http_reply: regexp statistics: 86 pcre regexps scanned, 3 regexps matched, 179 regexps total, 11 regexps cached, 10.55KiB scanned using pcre, 10.55KiB scanned total
2024-11-20 07:03:13 #453522(rspamd_proxy) <3ec2aa>; proxy; proxy_milter_finish_handler: finished milter connection
2024-11-20 07:04:11 #453522(rspamd_proxy) <01c02c>; proxy; proxy_accept_socket: accepted milter connection from ::1 port 44994
2024-11-20 07:04:45 #453522(rspamd_proxy) <103cbc>; proxy; proxy_accept_socket: accepted milter connection from ::1 port 58710

**Reply by moisseev** [reply]
Fine. You just need to read the log by yourself. It's written here in plain English:

> 2024-11-20 07:03:11 #453524(normal) <1C604C>; lua; settings.lua:391: [20241120060310.14025.qmail@cloudhost-10356613.uk-south-2.nxcli.net](mailto:20241120060310.14025.qmail@cloudhost-10356613.uk-south-2.nxcli.net) apply static settings global_wblist-14 (id = 3123186775); rcpt matched; priority high
> 2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action reject due to settings
> 2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action greylist due to settings
> 2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action rewrite subject due to settings
> 2024-11-20 07:03:11 #453524(normal) <1C604C>; task; lua_task_set_settings: disabled action add header due to settings

**Comment by Oxyash** [comment]
o myyy sorry, didnt see it
thak you so much

**Comment by Oxyash** [comment]
btw 
 "R_DUMMY symbol looks suspicious, as it suggests a user-configured rule."

what sound be score for this value  ? it wasnt configurated in other places
it was alwasy 0 as i can rmeber

**Reply by fatalbanana** [reply]
> btw "R_DUMMY symbol looks suspicious, as it suggests a user-configured rule."
> 
> what sound be score for this value ? it wasnt configurated in other places it was alwasy 0 as i can rmeber

I suppose it's copied from [here](https://groups.google.com/g/rspamd/c/8RqmIW9b5OM/m/mKe5L3KwTMYJ); maybe that's working & is fine.

Other approaches are possible (e.g. using redis+selector map in multimap).
